pyflakes/test/test_api.py

```diff
@@ -513,6 +513,7 @@ def test_invalidEscape(self):
             sourcePath, [decoding_error])
 
     @skipIf(sys.platform == 'win32', 'unsupported on Windows')
+    @skipIf(os.getuid() == 0, 'root can do anything')
```

Comment:
You need to move this inside the function then, e.g.,

```py
@skipIf(sys.platform == 'win32', 'unsupported on Windows')
def test_permissionDenied(self):
    """ ... """
    if os.getuid() == 0:
        self.skipTest('root can do anything')
    # ...
```

At that point, we'll be certain that we're not on Windows, whereas the module scope can't be certain.